xml/api/nodes-2.18.rng

```diff
@@ -16,6 +16,15 @@
             <attribute name="maintenance"> <data type="boolean" /> </attribute>
             <attribute name="pending"> <data type="boolean" /> </attribute>
             <attribute name="unclean"> <data type="boolean" /> </attribute>
+            <optional>
+                <attribute name="health">
+                    <choice>
+                        <value>red</value>
+                        <value>yellow</value>
+                        <value>green</value>
+                    </choice>
+                </attribute>
+            </optional>
```

Comment:
You're always adding health in node_xml, so this doesn't have to be optional - unless you want this schema to validate older stuff too.

Reply:
Yep, schemas should always be backward compatible (unless we bump the schema major version, which should be avoided)